modules/katello_content_view.py

```diff
@@ -188,8 +187,12 @@ def main():
     except Exception as e:
         module.fail_json(msg="Failed to connect to Foreman server: %s " % e)
 
+    kwargs = {}
+    if repositories:
+        kwargs['repositories'] = repositories
```

Comment:
We don't want to pass arguments if they're not given. It's different than passing `None`.